dashboard/db/migrate/20160815010139_add_view_answers_unlocked_at_to_user_levels.rb

```diff
@@ -0,0 +1,6 @@
+class AddViewAnswersUnlockedAtToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :view_answers, :boolean
+    add_column :user_levels, :unlocked_at, :datetime
```

Comment:
Regardless of column name choice, I'd encourage a comment here describing what the meaning of the column.